packages/react/src/analytics/integrations/Vitorino/providers/Riskified/Riskified.ts

```diff
@@ -0,0 +1,149 @@
+import {
+  EventData,
+  IntegrationFactory,
+  integrations,
+  LoadIntegrationEventData,
+  StrippedDownAnalytics,
+  TrackTypes,
+  TrackTypesValues,
+} from '@farfetch/blackout-analytics';
+import get from 'lodash/get';
+import type { RiskifiedIntegrationOptions } from './types';
+
+class Riskified extends integrations.Integration<RiskifiedIntegrationOptions> {
+  private isReady: boolean;
+  private siteId: string;
+  private baseUrl: string;
+
+  /**
+   * Creates an instance of Riskified and sets the script on the page.
+   *
+   * @param options - Custom options for the integration.
+   * @param loadData - Analytics's load event data.
+   */
+  constructor(
+    options: RiskifiedIntegrationOptions,
+    loadData: LoadIntegrationEventData,
+    analytics: StrippedDownAnalytics,
+  ) {
+    super(options, loadData, analytics);
+
+    this.isReady = false;
+    this.siteId = options.siteId || 'farfetch.com';
+    this.baseUrl = `https://beacon.riskified.com?shop=${this.siteId}&sid=`;
+
+    this.setupScript(loadData);
+  }
+
+  /* static createInstance<Options extends IntegrationOptions>(
+    this: IntegrationFactory<Options>,
+    options: Options,
+*/
+  static override createInstance<Options extends RiskifiedIntegrationOptions>(
```

Comment:
This method is not needed as the `Integration` class already does the same.

Reply:
Removed